### PR TITLE
Send button should be visible when the keyboard is open

### PR DESCRIPTION
**Pull request by DavDo**
MOB 2165

### CODE REVIEW

File: widgetssdk/src/main/java/com/glia/widgets/messagecenter/MessageCenterView.kt [hunk 1]
```diff
@@ -80,6 +87,21 @@ class MessageCenterView(
         // Is needed to overlap existing app bar in existing view with this view's app bar.
         ViewCompat.setElevation(this, 100.0f)
         readTypedArray(attrs, defStyleAttr, defStyleRes)
+
+        handleInstChanges()
+    }
+
+    private fun handleInstChanges() {
```
Comment (andrews-moc) [inline widgetssdk/src/main/java/com/glia/widgets/messagecenter/MessageCenterView.kt:94]: NIT: letter "e" is missing, isn't it? `handleInstChanges()` => `handleInsetChanges()`

File: widgetssdk/src/main/java/com/glia/widgets/helper/Insets.kt
Comment (andrews-moc) [inline widgetssdk/src/main/java/com/glia/widgets/helper/Insets.kt:1]: Honestly, I'm not sure I understand everything around the `Insets.kt`.
I think it makes sense to add comments to it, at least the following:
What is the purpose of `SimpleWindowInsetsAndAnimationHandler`?
What does it do?
When can the class and its functions be used?

File: widgetssdk/src/main/java/com/glia/widgets/helper/ContextExtensions.kt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.glia.widgets.helper
```
Comment (Pelkar) [inline widgetssdk/src/main/java/com/glia/widgets/helper/ContextExtensions.kt:1]: Maybe have a file called ViewExtensions, ContextExtensions is to specific to me

**Review by Pelkar — APPROVED** [review]
LGTM, one NIT

**Review by AndriiHorishniiMOC — APPROVED** [review]

**Comment by AndriiHorishniiMOC** [comment]
Looks good to me.

**Pull request merged**